test/functional/apps/discover/_shared_links.ts

```diff
@@ -88,7 +88,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
             '/app/discover?_t=1453775307251#' +
             '/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time' +
             ":(from:'2015-09-19T06:31:44.000Z',to:'2015-09" +
-            "-23T18:31:44.000Z'))&_a=(columns:!(),filters:!(),index:'logstash-" +
```

Comment:
So a shared link now contains `_source`? I think it would be better not to change this expectURL, but to revert the change that leads to the adding of `_source` to the shared link

Reply:
It contains `_source` right now. When switching to fields API, it won't include `_source`, as `_source` column doesn't exist anymore. Not sure if there's a proper fix for that. As for the test - I changed that test in master, this is basically reverting it to the current behavior.